README.md

```diff
@@ -103,8 +103,8 @@ Depending which tool you prefer to use (curl or httpie), use one of bash files a
 the address of the Red Hat Secured SSO Server and the Secured Vert.x Application.
 
 ```
-./scripts/httpie/token_req.sh https://secure-sso-sso.e8ca.engint.openshiftapps.com http://vertx-rest-sso.e8ca.engint.openshiftapps.com
-./scripts/curl/token_req.sh https://secure-sso-sso.e8ca.engint.openshiftapps.com http://vertx-rest-sso.e8ca.engint.openshiftapps.com
+./scripts/httpie/token_req.sh https://secure-sso-ssovertx.e8ca.engint.openshiftapps.com http://vertx-rest-sso.e8ca.engint.openshiftapps.com
+./scripts/curl/token_req.sh https://secure-sso-ssovertx.e8ca.engint.openshiftapps.com http://localhost:8080
```

Comment:
shouldn't this be `http://vertx-rest-sso.e8ca.engint.openshiftapps.com` instead of `http://localhost:8080`